project/Version.scala

```diff
@@ -1,25 +1,13 @@
 import sbt.Keys._
 import sbt._
 
-
 object Version {
-  val baseVersion = "0.1"
-
-  def getVersionString = {
-
+  private def getVersionString =
     // either specify git model hash as an env var or derive it
     // if building from the broadinstitute/scala-baseimage docker image use env var
     // (scala-baseimage doesn't have git in it)
-    val lastCommit = sys.env.getOrElse("GIT_HASH", sys.process.Process("git rev-parse --short HEAD")!! ).trim()
```

Comment:
Just so I understand: does this logic exist somewhere else or are we changing the format of the Leo version string?


Current version strings look like this: `0.1-971dc87-SNAPSHOT`

(see https://notebooks.firecloud.org/version)

Reply:
```
sys.env.getOrElse("GIT_HASH", sys.process.Process("git rev-parse --short HEAD")!! ).trim()
```
is still in place. We're just removing `0.1` and `SNAPSHOT` from the version string because they're constant and don't seem to add value. The git hash is already unique. 

Or did I misunderstand your question?

Reply:
OK thanks -- so the format of the version string is changing, from `0.1-<hash>-SNAPSHOT` to just `<hash>`. I agree the `0.1` and `SNAPSHOT` don't add any value. Just thinking if anything depends on the current version format, but I don't think anything does.

Reply:
Good thinking 👍  The only place I could think of was `hotswap` script but that seems to work.